PlantPal/PlantPal/View/ReusableView/CustomButton.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  CoustomButton.swift
+//  PlantPal
+//
+//  Created by Marcin Dytko on 09/07/2024.
+//
+
+import SwiftUI
+
+struct CustomButton: View {
+    var title: String
+    var action: () -> Void
+    
+    
+    var body: some View {
+        Button(action: {
+            self.action()
+        }) {
+            Text(title)
```

Comment:
Potrzebujemy jakiegoś wrappera na ten komponent. Dodałem zadanie, w którym powinniśmy zrobić wrapper na używanie czcionki + stworzenie wrappera na komponent tekstu. Możemy to obgadać na spotkaniu. 
https://mardytkos-team.monday.com/boards/1553207274/pulses/1556084326